api/tests/Unit/UserAuthorizationScopeTest.php

```diff
@@ -90,8 +86,6 @@ protected function setUp(): void
             ]);
     }
 
-    // no tests for scopeAuthorizedToViewSpecific since it is never directly used by the graphql schema
```

Comment:
Why did you remove this comment?

Reply:
It was an outdated comment: `scopeAuthorizedToViewSpecific` does not exist (even before this PR).